packages/cardano-services/test/jest-setup/snapshots/asset.sql

```diff
@@ -0,0 +1,1808 @@
+--
```

Comment:
Nice, generating this is part of LW-8695
Although commits probably have to be reordered, as some tests in previous commits of cardano-services are probably relying on changes of re-generated data.

Reply:
Moving 77aff56ba earlier in commits history should solve this